### DISCUSSION TITLE
怀疑 AList 与 V2Ray 冲突

### DISCUSSION THREAD

**Discussion by hans-johnson** [AlistGo/alist#6393] [Q&A]
我的 Chromebook，V2Ray（安装在 Android 子系统）正常时 AList 常常无法访问，AList 正常时 V2Ray 常常无法访问。
两个情况都发生过超过 1 个月的时间，并且无缘无故转换，譬如上月是 V2Ray 故障，AList 正常，找专家朋友来修无果，出门旅游闲置了 3 周，回来就翻转了。目前情况是 V2Ray 战胜 AList。

有时 V2Ray 断开，AList 立马就好，两者也同时正常过，但今年起就很少。尝试在多个设备（Android、Win 或本机 Linux 子系统），多个网络（公司、家）部署 AList，都是其他设备都正常访问，Chromebook 不行。

我为什么执着于用它看 AList，因为它的性能也就只能码字，但屏是 4K AMOLED。

**Comment by IAALAI** [comment]
你的alist不会是内网的吧，v2ray的代理有没有正确配置需要代理的站点？这都修不好，真的是专家吗？

**Reply by hans-johnson** [reply]
同一个 Wi-Fi；指的是分流吗？应该正确；毕竟也没几个人玩 chromebook😂
目前依然是只能一个开，另一个就得关，不然不正常的状态。
